docs/modules/ROOT/pages/cops_rspec_factorybot.adoc

```diff
@@ -151,6 +151,7 @@ This cop can be configured using the `EnforcedStyle` option
 ----
 # bad
 3.times { create :user }
+[create(:user), create(:user), create(:user)]
```

Comment:
I wonder if this style is acceptable in factories, though, as [FactoryBot lists this as an example](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#has_and_belongs_to_many-associations):
```ruby
factory :profile_with_languages do
  transient do
    languages_count { 2 }
  end

  languages do
    Array.new(languages_count) do
      association(:language, profiles: [instance])
    end
  end
end
```
And we [inspect factory definitions, too](https://github.com/rubocop/rubocop-rspec/blob/a0ef995bb02f604fbe45a56973046d21cb75c7b8/config/default.yml#L938).

`Array.new(count) { ... }` is more like `count.times.map { ... }`.